### DISCUSSION TITLE
Move this repo to a (new) GitHub organization

### DISCUSSION THREAD

**Discussion by stnolting** [stnolting/neorv32#234] [General]
This is a summary/work-in-progress/thoughts-gathering/you-name-it discussion triggered by PR #189 where the idea of turning this repository into an organization emerged.

Comments by @umarcor (please get in touch if I have missed some important points here):
 
> Therefore, I believe the most important decision at the moment is whether you are willing to create org neorv32 and transfer this repo there. From a permissions point of view, it would be the same (in fact better). As the owner of the organisation, you would be the only one with full access to all the repos. Moreover, organisations allow creating teams with different permission levels, while "regular" repos do only support one colaborator role. See and compare https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization and https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/permission-levels-for-a-user-account-repository.

> If you create the organisation, it makes sense to have potentially more than two repositories close to each other, maintained by different people. Conversely, if you prefer to stick with your user namespace, I would not recommend creating more than two repos. This is mostly because of visibility. You do have few repos in your account, so it would not be difficult to find them; but I think it is better to have an org from a "branding" perspective.

**Comment by umarcor** [comment]
Nitpicking... you cannot turn this repo into an organisation. You might turn *stnolting* into an organisation; but I believe you don't want to do that. What you would do is create a new organisation named *neorv32*, which you are the owner of. Then, as an owner of this repo and the organisation, you would *transfer* the repository. That would create the redirects so that people using the "old" URL gets to the "new" one.
